### PR TITLE
Reorganise Amino SignDoc and OfflineSigner

### PR DESCRIPTION
**Pull request by willclarktech**
Part of #657

### CODE REVIEW

File: packages/proto-signing/package.json [hunk 1]
```diff
@@ -44,7 +44,6 @@
   },
   "dependencies": {
     "@cosmjs/amino": "^0.25.0-alpha.0",
-    "@cosmjs/launchpad": "^0.25.0-alpha.0",
```
Comment (webmaster128) [inline packages/proto-signing/package.json:47]: 🎉

File: packages/amino/src/signer.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { StdSignature } from "@cosmjs/amino";
-
-import { StdSignDoc } from "./encoding";
+/* eslint-disable @typescript-eslint/naming-convention */
```
Comment (webmaster128) [inline packages/amino/src/signer.ts:1]: Why is this newly added here?

File: packages/launchpad/src/signingcosmosclient.ts [hunk 3]
```diff
@@ -1,13 +1,12 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+import { AminoMsg, Coin, makeSignDoc, OfflineAminoSigner, StdFee } from "@cosmjs/amino";
+import {} from "@cosmjs/amino/build/signdoc";
```
Comment (webmaster128) [inline packages/launchpad/src/signingcosmosclient.ts:3]: What happened here?

File: packages/amino/src/signer.ts (hunk 2 above)
Comment (willclarktech) [reply packages/amino/src/signer.ts:1]: Thanks, `StdSignDoc` was here for a part of the transition.

File: packages/launchpad/src/signingcosmosclient.ts (hunk 3 above)
Comment (willclarktech) [reply packages/launchpad/src/signingcosmosclient.ts:3]: Thanks, this is a remnant of autocomplete when I'd forgotten to export something.

**Review by webmaster128 — APPROVED** [review]
Could you rebase this to ensure it works in well together with my PR?

**Pull request merged**